ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java

```diff
@@ -43,7 +44,7 @@ private boolean isAppButtonPresent(String appId) {
     }
 
     public String getLoggedUsername() {
-        return findElement(By.cssSelector(".identifier")).getText();
+        return findElement(LOGGED_IN_USER).getText();
```

Comment:
Inspection on the `element` in context shows there are more than one element using the current value property. The suggested `attribute value property` uniquely identifies the element thus overriding.